static/includes/DisruptiveActionsList.md

```diff
@@ -0,0 +1,27 @@
+&NewLine;
+
+Configuring features and functions can result in disruption to services and access to users or system connecting to TrueNAS through sharing or network services.
+When adding new or changing existing functions and services, plan maintenance windows to minimize disruptions based on the type and area of change.
```

Comment:
This takes up a lot of screen real estate 
![image](https://github.com/user-attachments/assets/d04671b4-633c-481b-a5ec-659fa1458a2c)

I'm wondering if it would be better if all the text was within the expand

Reply:
I'd thought about that but decided the expand needed some introduction to help the reader decide if they want to view that list or just skip the list and plan based on the general areas mentioned, so I hid the list.